sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/CosmosRowConverter.scala

```diff
@@ -148,6 +150,17 @@ private[cosmos] class CosmosRowConverter(
       }
     }
 
+    def getChangeFeedLsn(objectNode: ObjectNode): String = {
+        getAttributeNode(objectNode, MetadataJsonBodyAttributeName) match {
+            case metadataNode: JsonNode =>
+                metadataNode.get(MetadataLsnAttributeName) match {
```

Comment:
I think this is identical to getAttributeAsString ?

Reply:
It is same, except that `getAttributeAsString()` only works on the top level json node. 
This gets the nested property. We can club them, but as I mentioned in the comments as well, some of these have been here to make code look cleaner and more readable.